starwars/src/components/Card.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Card = (props) => {
+
+    console.log(props)
+
+    return(
+    <div>{props.character}</div>
+    )}
```

Comment:
Should your card component display the individual values instead of the character object?